src/app/dim-ui/usePopper.ts

```diff
@@ -16,8 +16,8 @@ import computeSidecarPosition from 'app/item-popup/sidecar-popper-modifier';
 import _ from 'lodash';
 import React, { useLayoutEffect, useRef } from 'react';
 
-// ensure this stays in sync with 'arrow-size' in 'PressTip.m.scss'
-const popperArrowSize = 15;
+// ensure this stays in sync with '$theme-tooltip-arrow-size' in '_variables.scss'
+const popperArrowSize = 8;
```

Comment:
Since this value now reflects the arrow's size divided by two, we need to update the usage on line 41 to ensure that the arrow doesn't obscure the hovered element.

![dim-tooltip-arrow-placement](https://github.com/DestinyItemManager/DIM/assets/17512262/a799a841-ca59-439d-bc2b-e5b220998419)

https://github.com/DestinyItemManager/DIM/blob/4be7854f229d76108e3ebbff41545bc6f9abeaf4/src/app/dim-ui/usePopper.ts#L41

```diff
- offset = arrowClassName ? popperArrowSize / 2 : 0, 
+ offset = arrowClassName ? popperArrowSize : 0, 
```